python/in_toto_attestation/v1/resource_descriptor.py

```diff
@@ -19,12 +30,12 @@ def __init__(self, name: str='', uri: str='', digest: dict=None, content: bytes=
             self.pb.annotations.update(annotations)
 
     @staticmethod
-    def copy_from_pb(proto: type[rdpb.ResourceDescriptor]) -> 'ResourceDescriptor':
+    def copy_from_pb(proto: rdpb.ResourceDescriptor) -> "ResourceDescriptor":
```

Comment:
NB: I thought a bit about this, and I think this hint was wrong to begin with: `type[T]` is used an annotation for class objects themselves, which isn't what this API takes.